testing/config/settings/base.py

```diff
@@ -29,6 +29,10 @@
     'allauth.account'
 ]
 
+MIDDLEWARE += [
+    'allauth.account.middleware.AccountMiddleware'
```

Comment:
The test config using allauth in installed apps needs this middleware to run, otherwise django will not even start.